<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2595</width>
    <height>3284</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="MyWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>299</y>
     <width>250</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>250</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px ;
font-family: &quot;Verdana&quot;;
text-align: center;
color: white;
background-color: rgb(74, 98, 101,200);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>250</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px ;
font-family: &quot;Verdana&quot;;
text-align: center;
color: white;
background-color: rgb(74, 98, 101,0);
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Web-cam is busy&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_options">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/options.png</normaloff>:/res/options.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="MyWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>440</y>
     <width>380</width>
     <height>151</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="btn_start">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px ;
font-family: &quot;Verdana&quot;;
text-align: center;
color: black;</string>
    </property>
    <property name="text">
     <string>Launch</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>380</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px ;
font-family: &quot;Verdana&quot;;
text-align: center;
color: white;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Setup is complete&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(126, 147, 155);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>381</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(110, 129, 131);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>btn_start</zorder>
   <zorder>label_4</zorder>
   <zorder>label_3</zorder>
  </widget>
  <widget class="MyWidget" name="widg_options" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>411</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>19</y>
      <width>411</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px ;
font-family: &quot;Verdana&quot;;
text-align: center;
color: white;
background-color: rgb(110, 129, 131);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_9">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lbl_language">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: bold 13px ;
font-family: &quot;Verdana&quot;;
text-align: center;
color: white;</string>
       </property>
       <property name="text">
        <string>Language</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmb_lang">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>English</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Русский</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_13">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>160</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QLabel" name="lbl_webcam">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: bold 13px ;
font-family: &quot;Verdana&quot;;
text-align: center;
color: white;</string>
       </property>
       <property name="text">
        <string>Web-cam</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmb_webcam"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>411</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(126, 147, 155);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_close2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/x.png)stretch;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>129</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="lbl_notif">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font:  bold 13px;
font-family: &quot;Verdana&quot;;
text-align: center;
color: white;</string>
       </property>
       <property name="text">
        <string>Notifications</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkb_visual">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 14px ;
font-family: &quot;Verdana&quot;;
text-align: center;
color: white;</string>
       </property>
       <property name="text">
        <string>Visual</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkb_voice">
       <property name="styleSheet">
        <string notr="true">font-size: 14px ;
font-family: &quot;Verdana&quot;;
text-align: center;
color: white;</string>
       </property>
       <property name="text">
        <string>Sound</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="MyWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>640</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>480</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>640</width>
     <height>480</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(110, 129, 131);</string>
   </property>
   <widget class="QLabel" name="lbl_video">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px ;
font-family: &quot;Verdana&quot;;
text-align: center;
color: black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_title">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>640</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>640</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(94, 121, 131,200);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_close_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/x.png)stretch;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lbl_rules2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>640</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>640</width>
     <height>90</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px ;
font-family: &quot;Verdana&quot;;
text-align: center;
color: white;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyWidget</class>
   <extends>QWidget</extends>
   <header>mywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
